node.gypi

```diff
@@ -194,7 +194,18 @@
             },
           },
           'conditions': [
-            ['OS!="aix" and node_shared=="false"', {
+            ['OS=="linux" and node_shared=="false" \
```

Comment:
No need for `/`

Reply:
@refack Addressed in the latest updated PR.